DOL.WHD.Section14c.Web/src/modules/components/userLoginForm/userLoginFormDirective.test.js

```diff
@@ -0,0 +1,18 @@
+describe('userLogin', function() {
+
+    beforeEach(module('14c'));
+
+    var element, rootScope;
+    beforeEach(function () {
+        element = angular.element('<user-login/>');
+    	inject(function ($rootScope, $compile) {
+            rootScope = $rootScope;
+			$compile(element)(rootScope);
+			rootScope.$digest();
+        });
+    });
+
+    it('invoke directive', function() {
+        rootScope.$digest();
```

Comment:
updated directive unit tests to assert it was created https://github.com/AppliedIS/dol-whd-14c/commit/e73d8d8cc8291dbbcf47a7da435a7ad963cc54a4